.sites.config.yml

```diff
@@ -0,0 +1,10 @@
+---
+default:
+  theme_build:
+    - theme_path: src/Robo
```

Comment:
Can we provide a better example?

Reply:
Actually, how is this file even used?

Reply:
I feel like I'm missing something, as I see better examples in the file above.

Reply:
This file is present so we can [run tests in this repo](https://github.com/ChromaticHQ/chq-robo/blob/c91e5a9021499f8b284444add97303880914ff4c/.github/workflows/ci.yml#L50-L51) and verify the code is looking in the right place. The example file is what should be referenced for setting this up on a real site.